client/src/components/Workflow/List/WorkflowList.vue

```diff
@@ -79,7 +78,7 @@ const sharedWithMe = computed(() => props.activeList === "shared_with_me");
 const showDeleted = computed(() => filterText.value.includes("is:deleted"));
 const showBookmarked = computed(() => filterText.value.includes("is:bookmarked"));
 const currentPage = computed(() => Math.floor(offset.value / limit.value) + 1);
-const view = computed(() => (userStore.preferredListViewMode as ListView) || "grid");
+const currentListView = computed(() => userStore.currentListViewPreferences.workflows || "grid");
```

Comment:
I would rename `currentListView` to `currentListViewMode` to avoid confusion. The same for the similar computed in `ListHeader.vue` :)